renderer/src/components/ActivityChartCard.jsx

```
@@ -149,20 +135,20 @@
-                <FaBug size={12} />
+                <FaBug size={14} />
               </button>
               <button
                 onClick={() => {
                   setRetryCount(0);
                   generateChart(true);
                 }}
-                className="p-1 bg-gray-800 text-white rounded-full opacity-50 hover:opacity-100"
                 title="Force refresh chart"
+                className="p-2 bg-gray-800 text-white rounded-full opacity-75 hover:opacity-100 transition"
               >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)